### ISSUE TITLE
Bug when first argument is a string instead of an array of strings

### ISSUE THREAD

**Issue by jonaskuske** [ElMassimo/vite-plugin-full-reload#9]
Works:

```ts
fullReloadPlugin(['dir/*.php'])
```

Breaks:

```ts
fullReloadPlugin('dir/*.php')
```

The argument is passed to `Array.from`, so `dir/*.php` is turned into `['d', 'i', 'r', '/', '*', ...]`! Now all of these are watched as separate globs, including `*`, so lots of watcher processes are spawned, every single file in the project is watched and the development server slows to a crawl.

**Comment by ElMassimo** [comment]
It's not a bug, the plugin is expecting an Array.

If you have TypeScript in your editor it should warn you about it.

I'm ok accepting a pull request to throw an explicit error if it's not an Array.

**Comment by ElMassimo** [comment]
Confused it with another of my plugins.

Seems like `Array.from` is incorrect, I'll make a fix shortly.

**Comment by jonaskuske** [comment]
Thanks! :)

**Comment by ElMassimo** [comment]
Thanks for reporting, released a fix in `1.0.4` 😃
